apps/admin/components/github-signin.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { useEffect, useState, FC } from "react";
+import { AuthProviderButton } from "@repo/ui/components";
+
+import Link from "next/link";
+import { useSearchParams } from "next/navigation";
+import { cookieSetter } from "@/actions";
+import axios from "axios";
+import { GITHUB_SIGN_IN_URL } from "@/api-routes";
+
+type Props = {};
+
+export const GitHubSignInButton: FC<Props> = () => {
+  const clientId = process.env.NEXT_PUBLIC_GITHUB_CLIENT_ID;
+  const handleGitHubSignIn = async (credential: string) => {
+    try {
+      if (clientId && credential) {
+        const response = await axios.post(GITHUB_SIGN_IN_URL, {
+          code: credential,
+        });
+
+        cookieSetter({
+          access: response.data.tokens.access,
+          refresh: response.data.tokens.refresh,
+        });
+      } else throw Error("Cant find credentials");
+    } catch (err: any) {
+      // TODO: Handle error
+      console.error(err);
```

Comment:
```suggestion
      console.error(err);
```
please remove console